next/backend/services/ms-graph.ts

```diff
@@ -193,7 +212,7 @@ export const getUsersByDisplayName = async ({
   token,
   query,
 }: GetUsersByDisplayNameParams): Promise<MSGraphFilteredGroupUser[]> => {
-  const url = `https://graph.microsoft.com/v1.0/users?$search="displayName:${query}"`
+  const url = `https://graph.microsoft.com/v1.0/users?$select=otherMails,id,mail,displayName,businessPhones,jobTitle&$search="displayName:${query}"`
```

Comment:
Just a small detail, I would prefer to keep this attributes in same order as in `_.pick` function (on line 234), to be able to quickly check also visually if all attributes are present.